tests/unit/plugins/v1/test_waf.py

```diff
@@ -1,6 +1,6 @@
 # -*- Mode:Python; indent-tabs-mode:nil; tab-width:4 -*-
 #
-# Copyright (C) 2016-2018 Canonical Ltd
+# Copyright (C) 2016-2018, 2020 Canonical Ltd
```

Comment:
I see a mixed use of "," and "-".  Do you prefer `2018, 2020` or `2016-2020`?  Is your editor auto-updating it?

Not asking to change it, just curious

Reply:
Oh, 2016-2018 implies 2017 in the middle, for the cases where there were no changes in some years I resorted to `,`. And yes, I retroactively fixed the dates.